packages/component/src/SendBox/AttachmentBar/AttachmentBarItemStyle.ts

```diff
@@ -0,0 +1,104 @@
+import { StrictStyleOptions } from 'botframework-webchat-api';
+import { type StyleSet } from '../../Styles/StyleSet/types/StyleSet';
+
+export default function createSendBoxAttachmentBarItemStyle(_: StrictStyleOptions) {
+  return {
+    '&.webchat__send-box-attachment-bar-item': {
+      display: 'grid',
+      flexShrink: '0',
+      gridTemplateRows: 'auto',
+
+      '&.webchat__send-box-attachment-bar-item--as-list-item': {
+        borderRadius: '4px',
+        boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.14), 0px 0px 2px rgba(0, 0, 0, 0.12)',
+        gridTemplateAreas: '"body auxiliary"',
+        gridTemplateColumns: '1fr auto',
+        padding: '2px'
+      },
+
+      '&.webchat__send-box-attachment-bar-item--as-thumbnail': {
+        aspectRatio: '1/1',
+        border: 'solid 1px rgba(0, 0, 0, .25)', // Figma has border-width of 0.96px.
+        borderRadius: '8px', // Figma is 7.68px.
+        gridTemplateAreas: '"body"',
+        gridTemplateColumns: 'auto',
+        height: '80px', // <= 87px would fit white-label design with 3 thumbnails.
+        // height: '96px',
+        overflow: 'hidden'
+      }
+    },
+
+    // --- Delete button
```

Comment:
Maybe use region comments then? https://frontendmasters.com/blog/region-folding-in-vs-code/